### PR TITLE
Use placeholder attribute on metadata

### PR DESCRIPTION
**Pull request by jvoytek**

### CODE REVIEW

File: connectors/pme/pme_ui.js [hunk 1]
```diff
@@ -922,14 +922,17 @@ var SaveToFlow = (function() {
...
-							output.push("<div class='stf_lbl'>" + label + "</div><div class='stf_textposition'><span class='stf_empty'>Please enter metadata...</span>" + (field == "authors" ? "<span class='stf_author'>Last name, First names (each on a new line)</span>" : "") + "<textarea class='stf_val' id='stf_" + field + "' rows='1'></textarea></div>");
-						}
+						if (label.toLowerCase() == 'isbn')
+							value = value.split(',')[0];
+						output.push(
+							"<div class='stf_lbl'>" +
+								label +
+							"</div>" +
+							"<textarea class='stf_val' id='stf_" + field + "' rows='1' placeholder='" +
+							(field == "authors" ? "Last name, First names (each on a new line)" : "Please enter metadata...") +
```
Comment (aepheus) [inline connectors/pme/pme_ui.js]: problem with this is that the author placeholder isn't supposed to go away, it's supposed to trail after where the user is typing so they know how to add another author. so you'd have:

Jones, J
Last, first (each on a new line) <--- this never goes away.

Comment (jvoytek) [inline connectors/pme/pme_ui.js]: Does it matter that it wasn't doing that before? I'd say a much simpler solution would be to include some help text either below the field or next to the label.

Comment (aepheus) [inline connectors/pme/pme_ui.js]: That would be a viable alternative. The simpler the solution the better.

Comment (terrainsvagues) [inline connectors/pme/pme_ui.js]: Hey Jessica and Jeff! Miss chatting with you guys and hope all is well.  Can you remove me from this repo? 
Thanks! Nicky

Comment (aepheus) [inline connectors/pme/pme_ui.js]: ?I think you actually have to go into github and remove yourself. PME is open source, so anyone can be on it.

**Comment by vinht23** [comment]
This is ready to be merged into master.

**Comment by terrainsvagues** [comment]
OK I am unwatching this now. Thanks for the help, Caistarrin! And hi everyone...

**Comment by wspringer** [comment]
So am I ;-)

**Pull request merged**